### DISCUSSION TITLE
Refactor the docs

### DISCUSSION THREAD

**Discussion by helgeerbe** [hoylabs/OpenDTU-OnBattery#321] [Ideas]
We had a discussion lately in #313 regarding the docs.

What do you think, if we make use of the wiki? The wiki is like any other repo. So you can create PRs to change the documentation, if you would like to contribute. 

I would clean the README.md
- summarize onBattery key features
- summarize restrictions and extensions of upstream code
- reference the actual upstream version onBattery is build on
- link to onBattery change log
- link to onBattery wiki doc
- explain that general OpenDTU info is found in the upstream

**Comment by bf4ed** [comment]
Yes please. Clean the readme and make the wiki. The way it is here on GitHub with different files in docs folder is not practical. For first time user is confusing. Wiki is the better solution.

**Comment by helgeerbe** [comment]
I created the [wiki home page](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki) and set up a page structure. Wiki is open, so everybody is welcome to enhance the documentation.

In a next step I would replace the Readme with the wiki home page.

**Comment by helgeerbe** [comment]
Many Thanks to @Phantomias2006. He created the MQTT, Device Profiles, and Web Api pages in the wiki.
